GlobalSettings.php

```diff
@@ -36,6 +36,35 @@
 	};
 }
 
+// log suspicious or sensitive login attempts
+$wgHooks['AuthManagerLoginAuthenticateAudit'][] = static function ( $response, $user, $username ) {
```

Comment:
Should be done like other hooks with the wgHooks with others and the function done in MirahezeFunctions